### PR TITLE
Chore: Add StandardJS, remove ESLint

### PR DESCRIPTION
**Pull request by shaunorpen**
## Description
Add standard JS
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/ZcjQKmkE/21-setup-frontend#

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,6 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^6.7.2"
```
Comment (aapetsi) [inline package.json:45]: why did you remove eslint

Comment (shaunorpen) [reply package.json:45]: conflict with standard

Comment (aapetsi) [reply package.json:45]: okay. so you only catch the linting errors once you run the standard command @shaunorpen

**Review by aapetsi — APPROVED** [review]
Went through all the file changes and everything is good
